alkira/resource_alkira_credential_checkpoint.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2022 Alkira Inc. All Rights Reserved.
+package alkira
+
+import (
+	"log"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAlkiraCredentialCheckpoint() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCredentialCheckpoint,
+		Read:   resourceCredentialCheckpointRead,
+		Update: resourceCredentialCheckpointUpdate,
+		Delete: resourceCredentialCheckpointDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Description: "The name of the credential.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"password": &schema.Schema{
```

Comment:
It's probably good idea to support ENV var to set those... Eventually, no one wants to embed secrets in configs.